neqo-transport/src/server.rs

```diff
@@ -235,10 +236,20 @@ impl Server {
                             Output::None
                         },
                         |p| {
+                            qdebug!(
+                                [self],
+                                "type={:?} path:{} {}->{} {:?} len {}",
+                                PacketType::Retry,
+                                initial.dst_cid,
+                                dgram.destination(),
+                                dgram.source(),
+                                IpTos::default(),
```

Comment:
```suggestion
```
No need to log a constant.

Reply:
Logging it, because it's also part of the `dump_packet` output.